File: C#/7 kyu/0005.cs
// Kata: Largest Elements
// Link: https://www.codewars.com/kata/53d32bea2f2a21f666000256

// Write a program that outputs the top n elements from a list.

using System.Collections.Generic;

public class Kata
{
  public static List<int> Largest(int n, List<int> xs)
  {
     xs.Sort();
     return xs.GetRange(xs.Count - n, n);
  }
}

File: TypeScript/7 kyu/0005.ts
// Kata: Maximum Multiple
// Link: https://www.codewars.com/kata/5aba780a6a176b029800041c

// Task
//  - Given a Divisor and a Bound , Find the largest integer N , Such That ,

// Conditions :
//  - N is divisible by divisor
//  - N is less than or equal to bound
//  - N is greater than 0.

export function maxMultiple(divisor: number, bound: number): number {
  return bound - (bound % divisor);
}
